### ISSUE TITLE
Measuring tool

### ISSUE THREAD

**Issue by ctwhome** [NLeSC/zarrviz#13]
Adding the possibility to measure distances on the viewer with a rule or measuring tool from two selected points in the space.

**Comment by goord** [comment]
I would give this low priority, it was never explicitly requested in any of the meetings

**Comment by ctwhome** [comment]
Yes indeed, i put it under the nice to have status in the project
